app/src/main/kotlin/com/intive/picover/images/repository/ImagesRepository.kt

```diff
@@ -0,0 +1,41 @@
+package com.intive.picover.images.repository
+
+import android.net.Uri
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+import javax.inject.Inject
+import kotlin.coroutines.resumeWithException
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.suspendCancellableCoroutine
+import kotlinx.coroutines.withContext
+
+private const val REFERENCE_PATH = "image"
```

Comment:
Any reason to not just inline it 😄? It is used only in one place so no need to create val.